src/data/faq.json

```diff
@@ -101,6 +101,154 @@
+                        ]
+                    },
+                    {
+                        "title": "Am I obliged to use the digital certificate of vaccination in the Corona-Warn-App? Will there still be the yellow paper-based vaccination booklet? Can I continue to use my yellow vaccination booklet? What about people who don't own a smartphone?",
+                        "anchor": "vac_cert_voluntary",
+                        "active": true,
+                        "textblock": [
+                            "The digital proof of vaccination via the Corona-Warn-App is a completely voluntary and complementary offer. If you do not have a digital certificate of vaccination or have lost it, the proof of vaccination via the well-known 'yellow booklet' (yellow vaccination passport) is still possible and valid. You can also prove Corona vaccinations with the printed Corona vaccination certificate for full vaccination."                        ]
+                    },
+                    {
+                        "title": "How is the proof of vaccination verified?",
+                        "anchor": "vac_cert_verification",
+                        "active": true,
+                        "textblock": [
+                            "The vaccination certificate (QR code) is used digitally via the CovPass-App or the Corona-Warn-App (CWA) or alternatively as a machine-readable print. The vaccination certificate contains only information about the vaccination status, the name of the vaccinated person, and the date of birth. For service providers who want to check the vaccination status, there will be a verification app to check the vaccination certificate. This allows the vaccination status to be scanned in a similar way to a barcode of a flight or train ticket. Alternatively, proof with the paper-based vaccination document is possible."
+                        ]
+                    },
+                    {
+                        "title": "Why has no joint EU-wide project been called out for proposal for a certificate?",
+                        "anchor": "vac_cert_eu",
+                        "active": true,
+                        "textblock": [
+                            "A joint EU call for proposals would have taken too much time and would have been difficult to implement due to the different vaccination information systems in the Member States. The EU approach is about regulating a recognition framework. Any cross-border pilots could be accompanied by the EU Commission. When implementing the Digital Proof of Vaccination in Germany, the EU requirements have been into account from the outset."
+                        ]
+                    },
+                    {
+                        "titel": "Does my vaccination status affect the calculation of the risk? Does the calculation of the risk change if a vaccinated person has become infected, and then warns other users?",                        

```

Comment:
@dsarkar 
"titel" => "title"  !!
It would be helpful if you checked with an npm build before committing to the repository. Also would it be possible for you to bundle the commits? I find it almost impossible to keep up with the changes.

Reply:
ok!